.github/workflows/build.yml

```
@@ -9,6 +9,14 @@ jobs:
     runs-on: ubuntu-latest
     steps:
     - uses: actions/checkout@v3
+    - name: Install libtool v2.4.7
+      run: |
+        wget http://ftp.jaist.ac.jp/pub/GNU/libtool/libtool-2.4.7.tar.gz
```

```suggestion
        wget -q https://ftpmirror.gnu.org/libtool/libtool-2.4.7.tar.gz
```

It's kind of unfortunate this isn't getting cached, but for the expected frequency of use this is probably fine. The download taking 7 seconds and being hardwired to a specific JP mirror though doesn't seem right. How about using their mirror redirect?

---

Sounds good. I just picked a random mirror but I suppose that the main site will do better.